src/main/java/shop/service/impl/OrderServiceImpl.java

```diff
@@ -26,7 +28,8 @@ public Order getOrder(String orderId) {
     }
 
     @Override
-    public List<Order> getAllOrdersForUser(String usersId) {
+    public List<Order> getAllOrdersForUser() {
+        String usersId = UserSession.getInstance().getUser().map(User::getId).get();
```

Comment:
owww get() without checking? :)
Brave enough))

throw an exception if there is no logged user